tests/openssl_moretest/host/CSchema_checker.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "include_openssl.h"
+#include "string.h"
+
+#define O_ENCLAVE 0
+#define O_NATIVE 1
+
+#define RD_BUFF_SIZE (1 << 20)
+
+struct random_buffer
+{
+    char* buffer;
+    int first;
+    size_t size;
+    size_t native_indx;
+    size_t encl_indx;
```

Comment:
Style: Avoid abbreviations
```suggestion
    size_t native_index;
    size_t enclave_index;
```